### PR TITLE
Add back the security section from v1

### PR DESCRIPTION
**Pull request by pascalandy**
### Documentation fixes or enhancements:
- Docs fixes

### What does this PR do?

Docs: Add back the security section that was on the homepage from v1.7 docs

### Motivation

This docs was delete by accident when we merge from v1.7 to v2.0

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

That's my first PR over here :)

### CODE REVIEW

File: docs/content/contributing/submitting-security-issues.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Security
+
+## Security Advisories
+
+We strongly advise you to join our mailing list to be aware of the latest announcements from our security team. You can subscribe sending a mail to security+subscribe@traefik.io or on [the online viewer](https://groups.google.com/a/traefik.io/forum/#!forum/security).
```
Comment (ldez) [inline docs/content/contributing/submitting-security-issues.md]:
```suggestion
We strongly advise you to join our mailing list to be aware of the latest announcements from our security team.
You can subscribe sending a mail to security+subscribe@traefik.io or on [the online viewer](https://groups.google.com/a/traefik.io/forum/#!forum/security).
```

Comment (pascalandy) [reply docs/content/contributing/submitting-security-issues.md]: LGTM

**Comment by pascalandy** [comment]
Thanks!

**Comment by pascalandy** [comment]
Should I close this PR, you do you guys prefer to close it?

**Pull request merged**